test/parallel/test-net-socket-destroy-send.js

```diff
@@ -13,10 +13,18 @@ server.listen(0, common.mustCall(function() {
     // Test destroy returns this, even on multiple calls when it short-circuits.
     assert.strictEqual(conn, conn.destroy().destroy());
     conn.on('error', common.mustCall(function(err) {
```

Comment:
The `common.mustCall` is not necessary here. `common.expectsError` creates a function that can receive an error and also makes sure that it actually ran.